ocaml/xapi/helpers.ml

```diff
@@ -2282,13 +2282,19 @@ module AuthenticationCache = struct
 
     type session = Secret.secret
 
-    type t = {cache: Q.t; mutex: Mutex.t; elapsed: Mtime_clock.counter}
+    type t = {
+        cache: Q.t
+      ; mutex: Mutex.t
+      ; elapsed: Mtime_clock.counter
```

Comment:
Both entries could benefit from a comment. expiry is the maximum time an entry is staying in the cache before it it re-validated. Elapsed is?